neutrino.go

```diff
@@ -747,8 +753,7 @@ func NewChainService(cfg Config) (*ChainService, error) {
 	// peers and actively avoid advertising and connecting to discovered
 	// peers in order to prevent it from becoming a public test network.
 	var newAddressFunc func() (net.Addr, error)
-	if s.chainParams.Net != chaincfg.SimNetParams.Net &&
-		s.chainParams.Net != chaincfg.RegressionNetParams.Net {
+	if !isDevNetwork(s.chainParams.Net) {
```

Comment:
nit: can you update the comment above to indicate that we are no longer just switching on simnet here.

Reply:
Done.  Sorry about re-wrapping the comment.  It was starting to look wonky after making the change.